client/state/selectors/get-site-computed-attributes.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Hi @aduth , right now there is no get-site.js. The selector is in state/sites/selectors.js and If I export a named function in that file, I'm in fact creating a selector.
So as an alternative if you don't see any disadvantage I will add getSiteComputedAttributes to state/sites/utils.js or maybe get-site-utils.js. Creating an utils file, If we do that it may also make sense to move getJetpackComputedAttributes there because it is also only used in getSite and right now it is a selector (exported named function in the selectors file).